config/initializers/assets.rb

```diff
@@ -11,3 +11,4 @@
 # Rails.application.config.assets.precompile += %w( search.js )
 Rails.application.config.assets.precompile += %w( bubbles.js )
 Rails.application.config.assets.precompile += %w( bubbles_main.js )
+Rails.application.config.assets.precompile += %w( bubbles_main_google.js )
```

Comment:
You can pass them in, in the same collection separated by commas. In theory everything in the javascripts folder should be loaded if you have the `require_tree .` line in your application.js file